connect_four/ai.rb

```diff
@@ -0,0 +1,38 @@
+require './board.rb'
+
+class AI
+  attr_reader :player_id
+  attr_accessor :game_board
+
+  def initialize(player_id, game_board)
+    @player_id = player_id
+    @game_board = game_board
+  end
+
+  def pick_column
+    rational_choice = pick_rationally
```

Comment:
This is SO semantic... Or is the word explicit?... You know what I mean. Basically, my eyes love this right now.